fdbclient/BlobCipher.cpp

```diff
@@ -1707,10 +1707,9 @@ void testKeyCacheEssentials(DomainKeyMap& domainKeyMap,
 		Reference<BaseCipher> baseCipher = domainKeyMap[minDomainId][minBaseCipherKeyId];
 		uint8_t rawCipher[baseCipher->len];
 		memcpy(rawCipher, baseCipher->key.get(), baseCipher->len);
-		// modify few bytes in the cipherKey
-		for (int i = 2; i < 5; i++) {
-			rawCipher[i]++;
-		}
+		// modify cipherKey by flipping a bit
+		const int idx = deterministicRandom()->randomInt(0, baseCipher->len + 1);
```

Comment:
This missed the second commit to remove the +1 for some reason:

```suggestion
		const int idx = deterministicRandom()->randomInt(0, baseCipher->len);
```

Reply:
Ahh I usually "squash commit" into one keeping the documentary for history, seems like it went as two different commits in this case. Let me update the code